docs/provision-how-to.md

```diff
@@ -12,20 +12,10 @@ codeGetSubscriptionInfo: https://github.com/working-group-two/docs.wgtwo.com/blo
 ## Overview
 
 This API allows to enable and disable capabilities for a subscription and get subscription information.
+In order to access this API, your credentials need to have the `subscription.read` and `subscription.write` rights.
```

Comment:
I would prefer to keep the list of rights as a list, but not very important for me...

Reply:
Why?